src/sync/deployment.ts

```diff
@@ -66,7 +125,12 @@ export const getDeploymentTask = async (
 ) => {
 	logger.debug("Syncing Deployments: started");
 
-	const { edges, deployments } = await fetchDeployments(gitHubInstallationClient, repository, cursor, perPage);
+	const { edges, deployments, extraDeployments } = await fetchDeployments(jiraHost, gitHubInstallationClient, repository, logger, cursor, perPage);
+
+	const useDyanmoForBackfill = await booleanFlag(BooleanFlags.USE_DYNAMODB_FOR_DEPLOYMENT_BACKFILL, jiraHost);
+	if (useDyanmoForBackfill) {
```

Comment:
`useDyanmoForBackfill` to `useDynamoForBackfill`

Reply:
😞 crap. I'll leave this right now, hopefully the ff will be removed once 100% and this var will be removed in the followup PR.